### ISSUE TITLE
Panic when PostGIS query fails

### ISSUE THREAD

**Issue by dbazile** [terranodo/tegola#155]
Fat-fingered a SQL query in my `config.toml` and ended up referencing a column name that doesn't exist.  I tried throwing a few more bad queries at it (bogus table, syntax errors, etc) and it looks like any time a query fails for one reason or another, it will panic.

### Environment

Tegola Version: `0.3.2`
System: Debian Stretch (Docker)

### Stacktrace:

```
2017/10/24 00:47:20 http: panic serving 172.17.0.1:57474: runtime error: invalid memory address or nil pointer dereference
goroutine 281 [running]:
net/http.(*conn).serve.func1(0xc420ef6180)
	/home/travis/.gimme/versions/go1.7.5.linux.amd64/src/net/http/server.go:1491 +0x12a
panic(0x7cb260, 0xc42000c090)
	/home/travis/.gimme/versions/go1.7.5.linux.amd64/src/runtime/panic.go:458 +0x243
github.com/terranodo/tegola/mvt.(*Tile).AddLayers(0xc420171950, 0xc4201ca380, 0x3, 0x3, 0x9f6fe0, 0xc4210684e0)
	/home/travis/gopath/src/github.com/terranodo/tegola/mvt/tile.go:21 +0xf8
github.com/terranodo/tegola/server.HandleMapZXY.ServeHTTP(0xc42012433a, 0x5, 0x8, 0x45, 0x64, 0xc420124349, 0x3, 0x0, 0x9f6fe0, 0xc4210684e0, ...)
	/home/travis/gopath/src/github.com/terranodo/tegola/server/handle_map_zxy.go:164 +0x4a2
github.com/terranodo/tegola/server.(*HandleMapZXY).ServeHTTP(0xc420051080, 0x9f6fe0, 0xc4210684e0, 0xc4221f0960)
	<autogenerated>:4 +0x84
github.com/terranodo/tegola/vendor/github.com/dimfeld/httptreemux.(*ContextGroup).Handler.func1(0x9f6fe0, 0xc4210684e0, 0xc4221f0960, 0xc420124450)
	/home/travis/gopath/src/github.com/terranodo/tegola/vendor/github.com/dimfeld/httptreemux/context.go:62 +0x14f
github.com/terranodo/tegola/vendor/github.com/dimfeld/httptreemux.(*TreeMux).ServeHTTP(0xc420052d90, 0x9f6fe0, 0xc4210684e0, 0xc4221f0870)
	/home/travis/gopath/src/github.com/terranodo/tegola/vendor/github.com/dimfeld/httptreemux/router.go:252 +0x2e5
net/http.serverHandler.ServeHTTP(0xc420091000, 0x9f6fe0, 0xc4210684e0, 0xc4221f0870)
	/home/travis/.gimme/versions/go1.7.5.linux.amd64/src/net/http/server.go:2202 +0x7d
net/http.(*conn).serve(0xc420ef6180, 0x9f7820, 0xc4201446c0)
	/home/travis/.gimme/versions/go1.7.5.linux.amd64/src/net/http/server.go:1579 +0x4b7
created by net/http.(*Server).Serve
	/home/travis/.gimme/versions/go1.7.5.linux.amd64/src/net/http/server.go:2293 +0x44d
```

**Comment by ARolek** [comment]
@dbazile I'm pretty sure this has been fixed in the v0.4.0-beta version. You can download it from the [releases](https://github.com/terranodo/tegola/releases) page. Lots of new features in v0.4.0 including a built in viewer when you hit the server root (i.e. localhost:8080).

**Comment by dbazile** [comment]
@ARolek: confirmed fixed; `0.4.0-beta` fails fast on invalid queries at startup.  Thanks!
